homeassistant/components/switch/homematicip_cloud.py

```diff
@@ -34,9 +34,10 @@
     devices = []
     for device in home.devices:
         if isinstance(device, BrandSwitchMeasuring):
+            # BrandSwitch is implemented in the light platform
```

Comment:
Add why we need the case, ie about the inheritence of the classes. That's not obvious since the classes are implemented in a 3rd party library.